docs/lang/articles/reference.md

```diff
@@ -176,22 +176,72 @@ compile time.
 
 #### Calls
 
+```
+call          ::=  primary '(' argument_list ')'
+argument_list ::=  args
+args          ::=  [arg (',' arg)*]
+arg           ::=  identifier
+```
+
+To favor simplicity, Taichi language doesn't support keyword arguments like Python.
+
 ### The power operator
 
+```
+power ::= 'ti.pow(' primary ',' primary ')' | primary ['**' u_expr]
```

Comment:
`ti.pow` belongs to Taichi's builtin functions, so it doesn't have to be listed as a syntax rule here.